tests/integration/records/single_db/test_records_load_datetime.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import io
+import datetime
+from .base_records_test import BaseRecordsIntegrationTest
+from ..datetime_cases import (
+    DATE_CASES, DATETIMETZ_CASES, DATETIME_CASES, TIMEONLY_CASES, create_sample,
+    SAMPLE_YEAR, SAMPLE_MONTH, SAMPLE_DAY, SAMPLE_HOUR, SAMPLE_MINUTE, SAMPLE_SECOND
+)
+from records_mover.records import RecordsSchema, RecordsFormat, PartialRecordsHints
+from records_mover.records.schema.field.field_types import FieldType
+
+logger = logging.getLogger(__name__)
+
+
+class RecordsLoadDatetimeIntegrationTest(BaseRecordsIntegrationTest):
+
+    def load(self,
+             hint_name: str,
+             format_string: str,
+             column_name: str,
+             field_type: FieldType,
+             addl_hints: PartialRecordsHints = {}) -> None:
+        variant_for_db = {
+            'redshift': 'bluelabs',
+            'vertica': 'vertica',
+            'postgresql': 'bluelabs',
+            'mysql': 'bluelabs',
+            'bigquery': 'bigquery',
+        }
+        records_schema: RecordsSchema = RecordsSchema.from_data({
+            'schema': 'bltypes/v1',
+            'fields': {
+                column_name: {
+                    'type': field_type,
+                },
+            },
+        })
+        targets = self.records.targets
+        sources = self.records.sources
+        fileobj = io.BytesIO(create_sample(format_string).encode('utf-8'))
+        records_format = RecordsFormat(variant=variant_for_db[self.engine.name],
+                                       hints={
+                                           hint_name: format_string,  # type: ignore
+                                           'compression': None,
+                                           'header-row': False,
+                                           **addl_hints,
+                                       })
+        source = sources.fileobjs(target_names_to_input_fileobjs={
+                                    'test': fileobj
+                                  },
+                                  records_schema=records_schema,
+                                  records_format=records_format)
+        target = targets.table(schema_name=self.schema_name,
+                               table_name=self.table_name,
+                               db_engine=self.engine)
+        self.records.move(source, target)
+
+    def pull_result(self,
+                    column_name: str) -> datetime.datetime:
+        out = self.engine.execute(f'SELECT {column_name} '
+                                  f'from {self.schema_name}.{self.table_name}')
+        ret_all = out.fetchall()
+        assert 1 == len(ret_all)
+        ret = ret_all[0]
+        return ret[column_name]
+
+    # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker